crates/solver/src/liquidity_collector.rs

```diff
@@ -185,10 +185,10 @@ mod test {
         };
 
         let source =
-            BackgroundInitLiquiditySource::new("fake", init, Duration::from_millis(10), None);
+            BackgroundInitLiquiditySource::new("fake_delayed_init", init, Duration::from_millis(10), None);
```

Comment:
Not related to the ret of the PR, just fixes flakiness of this test when run in parallel with other uni tests.